packages/canary-web-components/src/components/ic-data-table/ic-data-table.types.tsx

```diff
@@ -40,6 +40,7 @@ export type IcDataTableColumnObject = {
     hideOnHeader?: boolean;
   };
   excludeColumnFromSort?: boolean;
+  visible?: boolean;
```

Comment:
Should this option be called `hidden` instead? Seems like all the examples talk about "hiding" columns, and the default behavior currently is that columns are visible unless otherwise specified. Feel like that exception should be to "hide" the column by setting it to true, that way leaving the option blank is the same as setting it to `false`, which I'd probably expect as a developer.

Reply:
I was torn as to the behaviour. I think you are correct though, `hidden` does make more sense - i'll make the change